Merge remote-tracking branch 'origin/master'

Conflicts:
	CoreConceptsPy/coreconcepts.py
#
# It looks like you may be committing a merge.
# If this is not correct, please remove the file
#	.git/MERGE_HEAD
# and try again.


# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
# On branch master
# Your branch and 'origin/master' have diverged,
# and have 2 and 4 different commits each, respectively.
#   (use "git pull" to merge the remote branch into yours)
#
# All conflicts fixed but you are still merging.
#
# Changes to be committed:
#	modified:   CoreConceptsPy/coreconcepts.py
#	modified:   CoreConceptsPy/examples/networks/data/ucsb.shp/ucsb.dbf
#	modified:   CoreConceptsPy/examples/networks/data/ucsb.shp/ucsb.shp
#	modified:   CoreConceptsPy/examples/networks/data/ucsb.shp/ucsb.shx
#	modified:   CoreConceptsPy/examples/networks/ucsb.py
#	modified:   CoreConceptsPy/networks.py
#	modified:   CoreConceptsPy/test/networks_test.py
#

